Source/WebInspectorUI/UserInterface/Views/FontDetailsPanel.js

```diff
@@ -130,58 +208,55 @@ WI.FontDetailsPanel = class FontDetailsPanel extends WI.StyleDetailsPanel
         this.element.appendChild(fontVariationPropertiesSection.element);
     }
 
-    _formatSizeValue(property)
-    {
-        return property.value;
-    }
-
-    _formatStyleValue(property)
+    _createDetailsSectionRowForProperty(propertyName)
     {
-        // `slnt` is expressed in negated values relative to `oblique`.
-        if (this._hasVariationValue(property, "slnt"))
-            return WI.UIString("Oblique %ddeg", "Oblique %ddeg @ Font Details Sidebar Property Value", "Property value for oblique text.").format(-1 * parseFloat(property.variations.get("slnt").value));
-
-        if (this._hasVariationValue(property, "ital"))
-            return WI.UIString("Italic", "Italic @ Font Details Sidebar Property Value", "Property value for oblique text.");
-
-        if (property.value === "normal")
-            return WI.UIString("Normal", "Normal @ Font Details Sidebar Property Value", "Property value for any `normal` CSS value.");
+        let fontProperty = this._fontPropertiesMap.get(propertyName);
+        let labelForTag = {
+            "ital": WI.UIString("Italic", "Italic @ Font Details Sidebar Property Value", "Property title for `font-style` italic and `ital` variation axis."),
+            "slnt": WI.UIString("Oblique", "Oblique @ Font Details Sidebar Property Value", "Property title for `font-style` oblique and `slnt` variation axis."),
+            "opsz": WI.UIString("Optical Sizing", "Optical Sizing @ Font Details Sidebar Property Value", "Property title for `font-optical-sizing` and `opzs` variation axis."),
+            "wght": WI.UIString("Weight", "Weight @ Font Details Sidebar Property", "Property title for `font-weight` and `wght` variation axis."),
+            "wdth": WI.UIString("Width", "Width @ Font Details Sidebar Property", "Property title for `font-stretch` and `wdth` variation axis."),
+        }
 
-        return property.value;
-    }
+        // TODO: Can there really be more than one variation axis mapped to a single property?
+        let fontVariationAxis = fontProperty.variations?.values().next().value;
+        if (fontVariationAxis) {
+            // Ensure registered axes have a name; fallback to labels for their corresponding font properties.
+            fontVariationAxis.name ||= labelForTag[fontVariationAxis.tag] || "";
```

Comment:
The name for the axis should be `null` from the backend if it isn't defined, and in general I think null is a better way to indicate "no value" than using an empty string, so I'd
```
fontVariationAxis.name ??= labelForTag[fontVariationAxis.tag];
```
And then handle the possibility that there may not be a name for an axis.

Reply:
The empty string fallback was intentional to make this selector and CSS rule match:
```
.details-section > .content > .group > .row.font-variation > .label:not(:empty) {
    grid-column: 1 / -1; /* The variation axis name, if present, spans the full width of the grid container. */
}
```

But I guess I can explicitly handle the missing name in `FontVariationDetailsSectionRow.js` and not generate the markup for it.